examples/flutter_gallery/lib/demo/rally/home.dart

```diff
@@ -0,0 +1,219 @@
+// Copyright 2019-present the Flutter authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import 'package:flutter/material.dart';
+import 'package:flutter/widgets.dart';
+import 'package:flutter_gallery/demo/rally/tabs/accounts.dart';
+import 'package:flutter_gallery/demo/rally/tabs/bills.dart';
+import 'package:flutter_gallery/demo/rally/tabs/budgets.dart';
+import 'package:flutter_gallery/demo/rally/tabs/overview.dart';
+import 'package:flutter_gallery/demo/rally/tabs/settings.dart';
+
+const int tabCount = 5;
+
+class HomePage extends StatefulWidget {
+  @override
+  _HomePageState createState() => _HomePageState();
+}
+
+class _HomePageState extends State<HomePage>
+    with SingleTickerProviderStateMixin {
+  _HomePageState() {
+    _tabController = TabController(length: tabCount, vsync: this);
+  }
+
+  TabController _tabController;
+
+  @override
+  void initState() {
+    super.initState();
+
+    _tabController.addListener(() {
+      if (_tabController.indexIsChanging &&
+          _tabController.previousIndex != _tabController.index) {
+        setState(() {});
```

Comment:
nit: setState(() { })

Reply:
usually empty functions in flutter/flutter are written as () { } instead of () {}